src/QuickMarcEditor/QuickMarcEditorRows/QuickMarcEditorRows.js

```diff
@@ -55,6 +73,30 @@ const QuickMarcEditorRows = ({ name, fields, mutators: { addRecord, deleteRecord
               data-test-quick-marc-editor-row
               data-testid="quick-marc-editorid"
             >
+              <div className={styles.quickMarcEditorMovingRow}>
+                {
+                  !withMoveUpRowAction && (
+                    <IconButton
+                      data-test-move-up-row
+                      data-index={idx}
+                      data-nearest-index={idx - 1}
```

Comment:
do you really need this attribute? you can get in function -1 or +1 based on direction

Reply:
@NikitaSedyx  In this case, I need 2 separate functions (to move up and move down). Do I have any better ways to know direction in one function?

Reply:
it's correct, 2 functions

Reply:
it's ok to have 1 function, `nearest-index` doesn't say a lot